### PR TITLE
server centralize request validation for co-location and reduced redundancy

### PR DESCRIPTION
**Pull request by dillonstreator**

### CODE REVIEW

File: packages/server/src/auth/newuser.ts [hunk 1]
```diff
@@ -31,12 +32,6 @@ export async function newUserHandler(req: Request, res: Response): Promise<void>
     return;
   }
 
-  const errors = validationResult(req);
-  if (!errors.isEmpty()) {
-    sendOutcome(res, invalidRequest(errors));
-    return;
-  }
-
```
Comment (dillonstreator) [inline packages/server/src/auth/newuser.ts:39]: Not sure if this approach is viable given that the behavior is slightly different here in that request validation would now run prior to the config check

**Comment by codyebberson** [comment]
Thanks @dillonstreator - this looks great.

**Comment by dillonstreator** [comment]
Thank you @codyebberson!
Please let me know if there is anything I can do to support this through the pipeline. I see some workflow failures but I believe they're outside of my control. This is lower priority than other work so no rush!

File: packages/server/src/auth/newuser.ts (hunk 1 above)
Comment (codyebberson) [reply packages/server/src/auth/newuser.ts:39]: It's ok 👍

**Review by codyebberson — APPROVED** [review]
Nice, thanks @dillonstreator !

Love this:
image

**Pull request merged**